templates/rack_mini_profiler.rb

```diff
@@ -0,0 +1,5 @@
+if ENV["RACK_MINI_PROFILER"].to_i > 0
```

Comment:
That change hasn't landed yet, and I don't believe this pull request is being held up. I just meant that configuring things using environment variables (as the pull request is currently doing) seems good to me, because it will work well when there are no separate staging/production environments.